verilog/tools/ls/autoexpand.cc

```diff
@@ -981,6 +981,12 @@ std::optional<AutoExpander::Expansion> AutoExpander::ExpandAutoinst(
     LOG(ERROR) << "AUTOINST: No definition found for module type: " << type_id;
     return {};
   }
+  if (NodeEnum(type_def->Tag().tag) != NodeEnum::kModuleDeclaration) {
+    LOG(ERROR) << "AUTOINST: Instance type " << type_id
+               << " is not a module, but a '" << NodeEnum(type_def->Tag().tag)
```

Comment:
So ideally, we should work towards that we can ask the symbol table 'do you have something with this name that is also module?'. Not sure if the symbol table well can keep track of these if they have multiple names, but that would be ideal.
@glatosinski do you know if that can be implemented in a simple way ?